webforj-plugins/webforj-install-bbj-plugin/pom.xml

```diff
@@ -29,6 +29,11 @@
       <artifactId>maven-invoker</artifactId>
       <version>3.2.0</version>
     </dependency>
+    <dependency>
+      <groupId>io.vavr</groupId>
+      <artifactId>vavr</artifactId>
+      <version>0.10.4</version>
+    </dependency>
```

Comment:
Where do you need this ? It does not seem it is used anywhere

Reply:
I'll remove it for now.  Some time ago i proposed that we use the maven assembly plugin to produce a fatjar, removing the need to store dependencies in the DWCJ plugin project.
This was refused.  Now ... it is being used.  Very curious that I was not told.
It will soon become time to refactor this project, to deal with the cyclomatic complexities.  Or not.  When that time comes I'll begin using this library.